test/Search_UnitTestHelpers.hpp

```diff
@@ -186,7 +186,7 @@ void checkResults(ArborX::DistributedSearchTree<DeviceType> const &tree,
   BOOST_TEST(indices_host == indices_ref, tt::per_element());
   BOOST_TEST(offset_host == offset_ref, tt::per_element());
   BOOST_TEST(ranks_host == ranks_ref, tt::per_element());
-  BOOST_TEST(distances_host != distances_ref, tt::per_element());
+  BOOST_TEST(distances_host == distances_ref, tt::per_element());
```

Comment:
How are both the new and the old version passing the tests? :'(

Reply:
I hope they are running... the whole thing is behind `#ifdef ARBORX_ENABLE_MPI`. If that does not work, all bets are off.

Reply:
What do you see if you just print the values?